app/views/revisions/show.html.erb

```diff
@@ -36,7 +36,7 @@
     </div>
   </div>
 
-  <pre><%= raw @diffed_revision.diff.format_as(:html) %></pre>
+  <pre class='diff-body'><%= @diffed_revision.diff.format_as(:color) %></pre>
```

Comment:
Since this is a JS-related class (i.e. no styling), can you prefix it accordingly? `js-diff-body`